server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -257,6 +257,18 @@ public void test_subscribing_to_unknown_publication_raises_error() throws Except
             .hasMessageContaining("Publication 'unknown_pub' unknown");
     }
 
+    @Test
+    public void test_subscribing_to_tables_with_higher_version_raises_error() throws Exception {
+        // TODO: create a table with older version
+        executeOnPublisher("CREATE TABLE doc.t1 (id INT) WITH(" + defaultTableSettings() + ")");
+
+        createPublication("pub1", false, List.of("doc.t1"));
+        assertThatThrownBy(() -> executeOnSubscriber("CREATE SUBSCRIPTION sub1" +
+            " CONNECTION '" + publisherConnectionUrl() + "' publication pub1"))
+            .isExactlyInstanceOf(IllegalStateException.class)
+            .hasMessageContaining("One of the published tables has version higher than subscriber's minimal node version.");
+    }
+
```

Comment:
We cannot do smth like CREATE TABLE  ... WITH(\"index.version.created\" = )", will create a unit tests with mocks instead and test this manually.